pkg/backupdriver/backup_driver_controller_base.go

```diff
@@ -798,6 +798,19 @@ func (ctrl *backupDriverController) syncDeleteSnapshotByKey(key string) error {
 		return err
 	}
 
+	if delSnapshot.Status.Phase == "" {
+		ctrl.logger.Infof("No Phase set for the DeleteSnapshot CR: %s/%s, will be explicitly update to New", namespace, name)
```

Comment:
nit: `will update to New`

Can you add a comment to explain why it needs to be updated to "New"?